### ISSUE TITLE
[1.7.10] NEI not displaying the upper left hot bar in the Inventory window.

### ISSUE THREAD

**Issue by llloyd4** [Chicken-Bones/NotEnoughItems#376]
So, recently I've been playing newer mod packs and I've been wanting to get back to an older version of Minecraft with some cool mods that are no longer around (like Thaumcraft). So I dusted off the The Ferret Business pack, it's kind of a mess but Caigan assures me it'll be updated soon. So I dusted off FTB Infinity, created a world with Cheats enabled, and threw it into Expert mode but no matter what I do, global overwrite or not global overwrite (W highlighted and not highlighted) on the hot bar, game mode, and cheat / recipe mode button the hot bar on the top left doesn't show up when I open my Inventory screen. It's the same with Ferret Business (existing world has Cheats enabled) and, for giggles, I installed FTB Crackpack for 1.7.10 (and made a Cheats enabled world) and all of the 1.7.10 packs I've poked at has this issue. I seem to remember there being a hot bar in the top left where I can toggle Creative / Adventure, Survival, etc, turn on and off the rain, etc. O.o

**Comment by llloyd4** [comment]
Update, just for those who are curious, in the NEI config file "Cheatmode=2" is set and "Lockmode=-1" is too so... It aught to be showing. O.o

**Comment by SoulVinicius** [comment]
no way! i think i finally found the problem by you...

**Comment by SoulVinicius** [comment]
my cheatmode was also set to 2 and my lockmode was also set to -1

**Comment by sc0rich** [comment]
Yeah, I'm having that same problem right now. I don't know how to fix it
